exporters/prometheus/config.go

```diff
@@ -132,9 +175,11 @@ func WithoutScopeInfo() Option {
 	})
 }
 
-// WithNamespace configures the Exporter to prefix metric with the given namespace.
-// Metadata metrics such as target_info are not prefixed since these
-// have special behavior based on their name.
+// WithNamespace configures the Exporter to prefix metric with the given
+// namespace. Metadata metrics such as target_info are not prefixed since these
+// have special behavior based on their name. Namespaces will be prepended even
+// if NoTranslation is set as a translation strategy. If the provided namespace
```

Comment:
```suggestion
// if [otlptranslator.NoTranslation] is set as a translation strategy. If the provided namespace
```

Reply:
if we want to set via string then this will probable change, I am guessing?